sdk/resourcemanager/azure-resourcemanager-cosmos/CHANGELOG.md

```diff
@@ -2,14 +2,23 @@
 
 ## 2.40.0-beta.2 (Unreleased)
 
-### Features Added
-
 ### Breaking Changes
 
-### Bugs Fixed
+#### `models.ServiceResourceCreateUpdateParameters` was modified.
+
+* `instanceCount()` was removed.
+* `instanceSize()` was removed.
+* `serviceType()` was removed.
+* `withInstanceCount(java.lang.Integer)` was removed.
+* `withInstanceSize(com.azure.resourcemanager.cosmos.models.ServiceSize)` was removed.
+* `withServiceType(com.azure.resourcemanager.cosmos.models.ServiceType)` was removed.
```

Comment:
- Properties under `ServiceResourceCreateUpdateParameters` class is moved to `ServiceResourceCreateUpdateProperties` class of its "properties" property.